### PR TITLE
dApp: Account menu redesign

### LINKED ISSUE #1509: Rework the account menu
## Description
Feedback within the demo
- Easier if the whole "button" would be clickable not just the small arrow

Personal remarks
- Show the new settings section also if connected but grayed out

Review
- Name of each action
- Description of each action

## Acceptance criteria
- Name of each action should be better
- Description of teach action should be better
- Group Raiden Account, Withdrawal and UDC
- Group Backup State and Report Bugs
- Make the whole button clickable

@sashseurat 

## Tasks
- [ ] Design @sashseurat 

## Current Screen

grafik

### PR DESCRIPTION
**Pull request by taleldayekh**
Fixes #1509 

**Short description**
This PR implements the reworked Account menu.
Screenshot 2020-11-06 at 11 55 52

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Navigate to the account menu

### CODE REVIEW

File: raiden-dapp/src/locales/en.json [hunk 1]
```diff
@@ -105,24 +105,24 @@
...
       "backup-state": {
-        "title": "Backup state",
-        "subtitle": "Download & upload your state"
+        "title": "Backup State",
+        "subtitle": "Download or upload your state"
       },
       "report-bugs": {
-        "title": "Report bugs",
+        "title": "Report Bugs",
         "subtitle": "Download logs"
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:113]:
```suggestion
        "subtitle": "Download your logs"
```

File: raiden-dapp/src/locales/en.json [hunk 2]
```diff
@@ -105,24 +105,24 @@
...
       },
       "raiden-account": {
-        "title": "Raiden Account",
-        "subtitle": "Transfer ETH between your main and Raiden account"
+        "title": "Manage Main/Raiden Account Funds",
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:116]:
```suggestion
        "title": "Transfer ETH",
```

File: raiden-dapp/src/locales/en.json [hunk 3]
```diff
@@ -105,24 +105,24 @@
...
       },
       "backup-state": {
-        "title": "Backup state",
-        "subtitle": "Download & upload your state"
+        "title": "Backup State",
+        "subtitle": "Download or upload your state"
       },
       "report-bugs": {
-        "title": "Report bugs",
+        "title": "Report Bugs",
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:112]:
```suggestion
        "title": "Download Logs",
```

File: raiden-dapp/src/locales/en.json (hunk 1 above)
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:113]:
```suggestion
        "subtitle": "Download your logs"
```

File: raiden-dapp/src/locales/en.json [hunk 4]
```diff
@@ -105,24 +105,24 @@
...
-        "title": "Raiden Account",
-        "subtitle": "Transfer ETH between your main and Raiden account"
+        "title": "Manage Main/Raiden Account Funds",
+        "subtitle": "Transfer ETH between main and Raiden account"
       },
       "withdrawal": {
-        "title": "Withdrawal",
-        "subtitle": "Withdraw tokens to your main account"
+        "title": "Withdraw Tokens",
+        "subtitle": "Withdraw tokens from Raiden account to main account"
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:121]:
```suggestion
        "subtitle": "Transfer tokens from your Raiden account to your main account"
```

File: raiden-dapp/src/locales/en.json [hunk 5]
```diff
@@ -105,24 +105,24 @@
...
       },
       "udc": {
-        "title": "UDC",
-        "subtitle": "Manage User Deposit Contract funds"
+        "title": "Manage UDC Tokens",
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:124]:
```suggestion
        "title": "Manage UDC",
```

File: raiden-dapp/src/locales/en.json [hunk 6]
```diff
@@ -105,24 +105,24 @@
...
+        "subtitle": "Manage User Deposit funds"
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:125]:
```suggestion
        "subtitle": "Transfer tokens from or to your User Deposit Contract"
```

**Review by christianbrb — COMMENTED** [review]
Changed the wording of the Menu items

**Review by weilbith — APPROVED** [review]
I reviewed this just from a technical perspective. I do not evaluate the design and text (as others did already evolved and evaluated it).
And the review is pretty short: looks pretty good! :grimacing:

**Pull request merged**